incubator/java-microprofile/stack.yaml

```diff
@@ -1,20 +1,24 @@
 name: Eclipse MicroProfile®
-version: 0.2.25
-description: Eclipse MicroProfile on Open Liberty & OpenJ9 using Maven
+version: 0.2.26
+description: This stack has been deprecated. Please use the Open Liberty stack. 
```

Comment:
I think we should not be changing the stack description as that gets pulled in by `appsody list` and also on appsody website. We should keep the deprecation message only on the `deprecated` field.

Reply:
Done